modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartDioClientCodegen.java

```diff
@@ -62,6 +61,7 @@ public DartDioClientCodegen() {
 
         outputFolder = "generated-code/dart-dio";
         embeddedTemplateDir = "dart-dio";
+        sourceFolder = "";
```

Comment:
can this be deduplicated into abstractdart?

Reply:
This client is being deleted on the other branch :)